yb-voyager/src/queryparser/ddl_processor.go

```diff
@@ -808,9 +808,69 @@ func (c *CreateType) GetObjectName() string {
 }
 func (c *CreateType) GetSchemaName() string { return c.SchemaName }
 
+//===========================VIEW PROCESSOR===================
+
+type ViewProcessor struct{}
+
+func NewViewProcessor() *ViewProcessor {
+	return &ViewProcessor{}
+}
+
+func (v *ViewProcessor) Process(parseTree *pg_query.ParseResult) (DDLObject, error) {
+	viewNode, ok := getCreateViewNode(parseTree)
+	if !ok {
+		return nil, fmt.Errorf("not a CREATE VIEW statement")
+	}
+	view := View{
+		SchemaName: viewNode.ViewStmt.View.Schemaname,
+		ViewName:   viewNode.ViewStmt.View.Relname,
+	}
```

Comment:
A comment for later:
```
viewMsg := GetMessageField(parseTreeMsg, "create_view")
if viewMsg == nil {
}

view := View {
       SchemaName: GetStringField(viewMsg, "schemaname")
       ViewName: GetStringField(viewMsg, "relname")
}

```
This way no need to write a specific function just for views like `getCreateViewNode()`

Reply:
Yeah, will start utilising it as we use these processor.

Reply:
Not doing these changes as of now.